paper-ripple-behavior.html

```diff
@@ -79,9 +79,11 @@
         }
         var domContainer = rippleContainer === this.shadyRoot ? this :
           rippleContainer;
-        if (opt_triggeringEvent &&
-            domContainer.contains(opt_triggeringEvent.target)) {
-          this._ripple.uiDownAction(opt_triggeringEvent);
+        if (opt_triggeringEvent) {
+          var target = opt_triggeringEvent.target;
+          if (domContainer.contains(/** @type {Node} */(target))) {
```

Comment:
can't we define the type of target in the function signature jsdoc above? doing it inline here feels super dirty